pkg/gui/controllers/commits_files_controller.go

```diff
@@ -41,6 +41,12 @@ func NewCommitFilesController(
 
 func (self *CommitFilesController) GetKeybindings(opts types.KeybindingsOpts) []*types.Binding {
 	bindings := []*types.Binding{
+		{
+			Key:         opts.GetKey(opts.Config.Files.CopyFileInfoToClipboard),
```

Comment:
I'm using a key from the Files section here, which could be considered wrong. Actually I think it's ok though, I prefer this both to adding a new one to the CommitFiles section, and to moving it into Universal. It's also not the first time we do this, we are already using a few others from the Files section here.